packages/create-liveblocks-app/scripts/demo.js

```diff
@@ -4,6 +4,34 @@ import { esbuildOptions } from "./esbuild.js";
 import fs from "fs";
 import path from "path";
 
+const pkgJson = `{
+  "name": "@liveblocks-examples/nextjs-comments",
+  "description": "This example shows how to build comments with Liveblocks and Next.js.",
+  "license": "Apache-2.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start"
+  },
+  "dependencies": {
+    "@liveblocks/client": "^1.11.0",
+    "@liveblocks/node": "^1.11.0",
+    "@liveblocks/react": "^1.11.0",
+    "@liveblocks/react-comments": "^1.11.0",
```

Comment:
Should this be `react-ui` or is that the entire goal here? To illustrate how it will get upgraded?

Reply:
It's a naive test so it doesn't matter too much, I'll remove it altogether to avoid confusion.